packages/eslint-config-twilio-react/package.json

```diff
@@ -42,9 +42,11 @@
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-no-only-tests": "^2.4.0",
+    "eslint-plugin-no-unsanitized": "^3.1.4",
     "eslint-plugin-prettier": "^3.3.0",
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-react-hooks": "^4.2.0",
+    "eslint-plugin-sonarjs": "^0.5.0",
```

Comment:
👀 isn't this already in eslint-config-twilio below?

Reply:
hmm ya you're right. @ktalebian do you rmb why we have some other plugins as deps here when they're already deps of `eslint-config-twilio` like `eslint-plugin-import` and `eslint-plugin-prettier`?

Reply:
o actually i think it's a lerna thing. i just tried removing it but lerna complained that it should be a dev-dependency

Reply:
It is a known lerna bug - setting up symlink for deps of deps does not work properly.

Reply:
👍 ahh that must be annoying :(